### PR TITLE
adding support for JWT SVID TTL 

### LINKED ISSUE #65: Add support to JWT SVID Ttl
Add support to JWT SVID TTL (jwt_svid_ttl) that  was introduced on [spire-api-sdk](https://github.com/spiffe/spire-api-sdk/commit/02f1e39e8856733f1353145d9b5812470f8619ec).

### PR DESCRIPTION
**Pull request by unnathik**
- added `JWTTTL` to `ClusterSPIFFEIDSpec`
- added `JWTTTL` to `renderPodEntry` func
- added test to check if entry JWT TTL matches the passed in JWT TTL
- closes #65

### CODE REVIEW

File: docs/clusterspiffeid-crd.md [hunk 1]
```diff
@@ -21,7 +21,8 @@ The definition can be found [here](../api/v1alpha1/clusterspiffeid_types.go).
 | `namespaceSelector`         | OPTIONAL | A label selector used to scope which workload namespaces this ClusterSPIFFEID targets |
 | `dnsNameTemplates`          | OPTIONAL | One or more templates used to render DNS names for the target workload. See [Templates](#templates). |
 | `workloadSelectorTemplates` | OPTIONAL | One or more templates used to render additional selectors for the target workload. See [Templates](#templates). |
-| `ttl`                       | OPTIONAL | Duration value indicating an upper bound on the time-to-live for SVIDs issued to target workload |
+| `ttl`                       | OPTIONAL | Duration value indicating an upper bound on the time-to-live for X509-SVIDs issued to target workload |
+| `jwtTtl`                     | OPTIONAL | Duration value indicating an upper bound on the time-to-live for JWT-SVIDs issued to target workload |
```
Comment (MarcosDY) [inline docs/clusterspiffeid-crd.md:25]: can you solve this indentation issue?
```suggestion
| `ttl`                       | OPTIONAL | Duration value indicating an upper bound on the time-to-live for X509-SVIDs issued to target workload |
| `jwtTtl`                    | OPTIONAL | Duration value indicating an upper bound on the time-to-live for JWT-SVIDs issued to target workload |
```

**Review by MarcosDY — COMMENTED** [review]
Looks great! just a minor comment

**Review by MarcosDY — DISMISSED** [review]
LGTM!

File: pkg/spireentry/entries_test.go [hunk 2]
```diff
@@ -3,6 +3,8 @@ package spireentry
 import (
 	"testing"
 
+	"time"
```
Comment (faisal-memon) [inline pkg/spireentry/entries_test.go:6]:
```suggestion
	"testing"
	"time"
```

**Review by MarcosDY — APPROVED** [review]

**Comment by drewwells** [comment]
Sweet! I didn't fully read how this is integrated. Does this override the ttl flags passed in the config to controller manager?

**Pull request merged**